### ISSUE TITLE
How to use CogVLM in image caption tasks.

### ISSUE THREAD

**Issue by sanbuphy** [THUDM/CogVLM#24]
Could you please provide me with some guidance on how to employ CogVLM when it comes to fulfilling image captioning tasks? Maybe we don't need the chat version

**Comment by 1049451037** [comment]
Just use the chat version and make a query "Describe the image."

**Comment by sanbuphy** [comment]
> Just use the chat version and make a query "Describe the image."

Yes, I am currently using this method to get the caption answer, but it need so much gpu compute resourse, maybe i should wait for the int4 version

**Comment by Blankit** [comment]
部署的时候要占多少显存啊

**Comment by zzhanq** [comment]
How to use CogVLM in image caption tasks with batch? Only batch_size=1 is supported?

**Comment by pf67** [comment]
> 部署的时候要占多少显存啊

用了4张4090跑起来的

**Comment by chakrabortyrajatsubhra** [comment]
Can somebody guide me how to do image captioning task with CogVLM like I have a folder of images and one fixed query like "Describe the image" and I need to save the captions, how do I do that with the cli_sat model in the demo, or do I need to follow some other way? Any help would be appreciated.

**Comment by ProGamerGov** [comment]
@chakrabortyrajatsubhra Checkout my scripts here for how to caption a directory of images with CogVLM: https://github.com/ProGamerGov/VLM-Captioning-Tools/tree/main
